test/e2e/storage/csi_mock_volume.go

```diff
@@ -772,17 +772,23 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 
 		createVolume := "CreateVolume"
 		deleteVolume := "DeleteVolume"
-		publishVolume := "NodePublishVolume"
+		// publishVolume := "NodePublishVolume"
 		unpublishVolume := "NodeUnpublishVolume"
-		stageVolume := "NodeStageVolume"
+		// stageVolume := "NodeStageVolume"
 		unstageVolume := "NodeUnstageVolume"
 
 		// These calls are assumed to occur in this order for
-		// each test run.
+		// each test run. NodeStageVolume and
+		// NodePublishVolume should also be deterministic and
```

Comment:
Can we open an issue to track this problem?

Reply:
https://github.com/kubernetes/kubernetes/issues/91155